samples/boards/gd32/i2c_at24c02c/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _gd32_i2c_at24c02c:
+
+I2C AT24C02C
```

Comment:
note: Zephyr has a driver for this EEPROM: https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/eeprom/eeprom_at2x.c, you should be able to verify with https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/drivers/eeprom

Reply:
done.